deisctl/backend/fleet/stop.go

```diff
@@ -94,6 +94,7 @@ func doStop(c *FleetClient, target string, wg *sync.WaitGroup, out, ew io.Writer
 			fmt.Fprintf(ew, o, target)
 			return
 		}
+		fmt.Println(lastSubState)
```

Comment:
Does this fix something not specific to the PR? I thought `deisctl stop` worked fine...